docs/content/en/docs/use-cases/process-lifecycle/advanced-process-execution.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Advanced Process execution"
+weight: 2
+icon: "overview"
+description: "Advanced Process Execution monitoring"
+---
+
+## Monitor Elf or Flat binaries execution
+
+Advanced process execution can be performed by using [Tracing Policies](/docs/concepts/tracing-policy) to monitor
+the [execve system call](https://man7.org/linux/man-pages/man2/execve.2.html) path.
+
+If we want to monitor execution of Executable and Linkable Format (ELF) or flat binaries
+before they are actually executed. Then the [process-exec-elf-begin](https://github.com/cilium/tetragon/blob/main/examples/tracingpolicy/process-exec/process-exec-elf-begin.yaml) tracing policy is a good first choice.
+
+{{< note >}}
+Note: the [process-exec-elf-begin](https://raw.githubusercontent.com/cilium/tetragon/main/examples/tracingpolicy/process-exec/process-exec-elf-begin.yaml) tracing policy, will not report the
+different binary format handlers or scripts being executed, but will report
+the final elf or flat binary, like the shebang handler.
+
+To report those another tracing policy can be used.
+{{</ note >}}
+
+Before going forward, verify that all pods are up and running, ensure you
+deploy our Demo Application to explore the Security Observability Events:
+
+```bash
+kubectl create -f https://raw.githubusercontent.com/cilium/cilium/v1.11/examples/minikube/http-sw-app.yaml
+```
+
+It might take several seconds for some pods until they satisfy all the dependencies:
+
+```bash
+kubectl get pods -A
+NAMESPACE            NAME                                         READY   STATUS    RESTARTS   AGE
+default              deathstar-54bb8475cc-6c6lc                   1/1     Running   0          2m54s
+default              deathstar-54bb8475cc-zmfkr                   1/1     Running   0          2m54s
+default              tiefighter                                   1/1     Running   0          2m54s
+default              xwing                                        1/1     Running   0          2m54s
+kube-system          tetragon-sdwv6                               2/2     Running   0          27m
+```
```

Comment:
[separate command and output](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output)
````suggestion
```shell-session
kubectl get pods -A
```

The output should be similar to:

```text
NAMESPACE            NAME                                         READY   STATUS    RESTARTS   AGE
default              deathstar-54bb8475cc-6c6lc                   1/1     Running   0          2m54s
default              deathstar-54bb8475cc-zmfkr                   1/1     Running   0          2m54s
default              tiefighter                                   1/1     Running   0          2m54s
default              xwing                                        1/1     Running   0          2m54s
kube-system          tetragon-sdwv6                               2/2     Running   0          27m
```
````